piccolo_api/session_auth/templates/signup.html

```diff
@@ -0,0 +1,32 @@
+{% extends "base.html" %}
+
+{% block title %}Signup{% endblock %}
+
+{% block content %}
+<h1>Signup</h1>
+
+{% if error %}
+<p class="error">{{ error }}</p>
+{% endif %}
+
+<form method="POST">
+    <label>Username</label>
+    <input type="text" name="username" required />
+    <label>Email</label>
+    <input type="email" name="email" required />
+    <label>Password (6 characters minimum)</label>
+    <input type="password" name="password" minlength=6 required />
+    <label>Confirm Password (6 characters minimum)</label>
```

Comment:
I don't think we need to repeat the `(6 characters minimum)`.

Reply:
You're right. I'll remove that.